src/fp6.rs

```diff
@@ -142,6 +142,20 @@ impl Fp6 {
         }
     }
 
+    /// Returns whether or not this element is strictly lexicographically
+    /// larger than its negation.
+    #[inline]
+    pub fn lexicographically_largest(&self) -> Choice {
+        // If this element's c1 coefficient is lexicographically largest
+        // then it is lexicographically largest. Otherwise, in the event
+        // the c1 coefficient is zero and the c0 coefficient is
+        // lexicographically largest, then this element is lexicographically
+        // largest.
+
+        self.c1.lexicographically_largest()
```

Comment:
This cannot be correct given that Fp6 contains c2, right?

Reply:
Oh my, of course.